### PR TITLE
Use viperHTML to statically build the site

### PR DESCRIPTION
**Pull request by ExE-Boss**
Since #21 was rejected with the rejection reason given in https://github.com/viperHTML/viperhtml.github.io/pull/21#issuecomment-397205588:
> I appreciate this effort but it would be silly to use anything different from basicHTML or viperHTML to statically pre-generate such content.
>
> This whole project is about template literals to create HTML, using third parts to achieve the same goal would make this project look useless.
>
> Sorry for not accepting this PR and thanks for your understanding.

I went ahead and setup a build system to generate the website content using viperHTML (I’ll probably eventually also add a gulp command [using Browsersync](https://www.browsersync.io/) for easier local testing).

## How to merge
1. Create a `src` branch and make it the new default branch (`*.github.io` user/organisation sites can only be deployed from the `master` branch, not from the `gh‑pages` branch, like project GitHub Pages sites can)
2. Change the base branch of this repository into the newly created `src` branch
3. Merge this into the `src` branch
4. Install [Travis CI](https://github.com/apps/travis-ci) into this repository
5. Make a commit replacing the `GH_TOKEN` environment variable with an actual GitHub deploy token following [this guide](https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml): https://github.com/viperHTML/viperhtml.github.io/blob/a4cf3e3271a3d94651305aa0385fc1c479e94857/.travis.yml#L4 and push the commit to the `src` branch (make sure to use the `api.travis-ci.com` API endpoint to get the encryption keys and not the legacy `api.travis-ci.org` API endpoint, otherwise Travis will fail to decrypt the `GH_TOKEN`).

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    es6: true,
+    node: true,
+  },
+  extends: 'eslint:recommended',
+  parserOptions: {
+    ecmaVersion: 2018,
+  },
+  root: true,
+  rules: {
+    eqeqeq: 'error',
```
[the remark this answers is not in the document]
Comment (ExE-Boss) [reply .eslintrc.js]: Done

**Comment by ExE-Boss** [comment]
Also, I told you in the instructions to install Travis CI as a GitHub App to support [the latest Travis features](https://docs.travis-ci.com/user/open-source-on-travis-ci-com/#New-Repositories).

**Comment by WebReflection** [comment]
ph crap ... I think everything ever I am using n GitHub with travis is from legacy .org version ... kill me now.

**Comment by WebReflection** [comment]
> Also, I told you in the instructions to install Travis CI as a GitHub App

current status ...

screen shot 2018-07-04 at 23 08 21

**Comment by ExE-Boss** [comment]
Well, Travis will be migrating old repositories to https://travis-ci.com eventually, hopefully before Half‑Life 3 comes out.

**Comment by WebReflection** [comment]
gotta go to sleep now, hoping tomorrow they have migrated this (sure ... )

**Comment by ExE-Boss** [comment]
You could at least merge this now that I fixed the build.

**Comment by WebReflection** [comment]
> You could at least merge this now that I fixed the build.

if I merge this into master the whole site will be gone

**Comment by WebReflection** [comment]
oh ... actually, you are targeting src now

**Comment by ExE-Boss** [comment]
I already changed the base branch to `src`:
image

**Pull request merged**